packages/cli/tests/e2e/m365/DebugM365Tab.tests.ts

```diff
@@ -34,65 +30,72 @@ describe("Debug V3 m365-tab template", () => {
     if (!isV3Enabled()) {
       this.skip();
     }
-
-    // clean up
-    const context = await readContextMultiEnvV3(projectPath, "local");
-    if (context?.TEAMS_APP_ID) {
-      await deleteTeamsApp(context.TEAMS_APP_ID);
-    }
-    if (context?.AAD_APP_OBJECT_ID) {
-      await deleteAadAppByObjectId(context.AAD_APP_OBJECT_ID);
-    }
-    await cleanUpLocalProject(projectPath);
+    await Cleaner.clean(projectPath);
```

Comment:
Thank you for the contribution. Will you apply this utility to all other cases, or should the owners do that?

Reply:
I just apply some cases to verify the tools for now. Others may be applied later by me or owners or testers. That is not decided yet.